src/main/java/practice/StreamPractice.java

```diff
@@ -35,7 +54,11 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * Example: select men who can be recruited to army (from 18 to 27 years old inclusively).
      */
     public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(p -> p.getSex().equals(Person.Sex.MAN)
+                        && (p.getAge() >= fromAge
+                        && p.getAge() <= toAge))
```

Comment:
Do you actually need those brackets?
```suggestion
                        && p.getAge() >= fromAge
                        && p.getAge() <= toAge)
```

Reply:
no))